docs/azure-monitor-logging.md

```diff
@@ -0,0 +1,81 @@
+# Azure Monitor tagging
+
+All telemetry sent to Azure Monitor, whether traces, metrics, or spans, should follow this guide.
+
+## Overloading of default dimensions
+
+A few built-in Azure Monitor fields are overloaded:
+
+| field | overload |
+| - | - |
+| `cloud_RoleName` | Either `device` or `service` |
+| `cloud_RoleInstance` | `runId` for the process producing telemetry. |
+
+## customDimensions
+
+```json
+  {
+    "sdkLanguageVersion": "3.8.2",
+    "sdkLanguage": "python",
+    "sdkVersion": "2.2.3",
+    "lineNumber": "291",
+    "sessionId": "a8b42042-6a08-4bcd-85c3-3d96d25ccd64",
+    "fileName": "service.py",
+    "deviceId": "bertk_test_device_4",
+    "process": "MainProcess",
+    "module": "service",
+    "osType": "Linux",
+    "poolId": "bertk_desktop_pool",
+    "runId": "3c4d68ed-e8ac-422e-85c6-5937a92c1c43",
+    "level": "INFO",
```

Comment:
In azure monitor, logging level is another field. Do we want it in custom properties too?

Reply:
python specific.  added automatically via the OpenCensus wrappers that we use for Azure Monitor.  moving down in the doc